Demo/Common/IntegrationExamples/AdvancedFlowExample.swift

```diff
@@ -0,0 +1,374 @@
+////
+//// Copyright (c) 2023 Adyen N.V.
+////
+//// This file is open source and available under the MIT license. See the LICENSE file for more info.
+////
+//
+
+import AdyenActions
+import AdyenComponents
+import AdyenDropIn
+
+internal final class AdvancedFlowExample: AdvancedFlowExampleProtocol {
+
+    internal var advancedFlowComponent: PresentableComponent?
+    internal var paymentMethods: PaymentMethods?
+    internal weak var presenter: PresenterExampleProtocol?
+
+    internal lazy var adyenActionComponent: AdyenActionComponent = {
```

Comment:
One don't need `AdyenActionComponent` for Drop-In

Reply:
Removed it. da858bb9a858774fa139005a65bb215f94ed0c10